web/styles/inbox.css

```diff
@@ -451,6 +451,26 @@
     .hidden_by_filters {
         display: none !important;
     }
+
+    .channel-folders-inbox-menu-icon {
+        display: grid;
+        /* width excluding left & right margin */
+        width: calc(var(--left-sidebar-vdots-width) - 3px);
+        cursor: pointer;
+        place-items: center center;
+        border-radius: 3px;
+        margin: 2px 2px 2px 1px;
+        color: var(--color-vdots-visible);
+
+        & i.zulip-icon {
```

Comment:
This selector should be `channel-folders-inbox-menu-icon`, right?

Reply:
Nope, but it was indeed not applying. Fixed like this:

``` diff
diff --git a/web/styles/inbox.css b/web/styles/inbox.css
index 4c6d6a9f38..7aee3ee6cf 100644
--- a/web/styles/inbox.css
+++ b/web/styles/inbox.css
@@ -462,7 +462,7 @@
         margin: 2px 2px 2px 1px;
         color: var(--color-vdots-visible);
 
-        & i.zulip-icon {
+        i.zulip-icon {
             font-size: 1.0625em;
         }
 
```

Reply:
Thanks!